definitions/divorce/json/CaseEventToFields/CaseEventToFields-payment-by-account-nonprod.json

```diff
@@ -172,39 +185,243 @@
     "LiveFrom": "02/10/2020",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorStatementOfTruthPaySubmit",
-    "CaseFieldID": "SolicitorFeeAccountNumber",
-    "PageFieldDisplayOrder": 3,
+    "CaseFieldID": "D8HelpWithFeesReferenceNumber",
+    "PageFieldDisplayOrder": 5,
     "DisplayContext": "MANDATORY",
     "PageID": "SolPayment",
     "PageLabel": "Payment",
     "PageDisplayOrder": 2,
-    "FieldShowCondition": "SolPaymentHowToPay=\"feePayByAccount\"",
+    "FieldShowCondition": "SolPaymentHowToPay=\"feesHelpWith\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
-    "LiveFrom": "02/10/2020",
+    "LiveFrom": "05/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "PbaNumbers",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "SolPayAccount",
+    "PageLabel": "Pay account",
+    "PageDisplayOrder": 3,
+    "PageShowCondition": "SolPaymentHowToPay=\"feePayByAccount\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "05/10/2020",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorStatementOfTruthPaySubmit",
     "CaseFieldID": "FeeAccountReference",
-    "PageFieldDisplayOrder": 4,
+    "PageFieldDisplayOrder": 2,
     "DisplayContext": "MANDATORY",
-    "PageID": "SolPayment",
-    "PageLabel": "Payment",
-    "PageDisplayOrder": 2,
-    "FieldShowCondition": "SolPaymentHowToPay=\"feePayByAccount\"",
+    "PageID": "SolPayAccount",
+    "PageLabel": "Pay account",
+    "PageDisplayOrder": 3,
     "ShowSummaryChangeOption": "Yes"
   },
   {
-    "LiveFrom": "02/10/2020",
+    "LiveFrom": "01/01/2017",
```

Comment:
Could you update the date on this?

Reply:
and subsequent ones please

Reply:
Oh actually they do have a change in the page display order yep 🙂 👍

Reply:
Hmm, well actually I'm wondering if it would be good to leave the date like this so we know that those fields were there a long time, as we just modified the page display order